src/core/common/instance.hpp

```diff
@@ -745,6 +750,13 @@ template <> inline DuaManager &Instance::Get(void)
 }
 #endif
 
+#if (OPENTHREAD_MTD || OPENTHREAD_FTD) && OPENTHREAD_CONFIG_MLR_ENABLE
+template <> inline MlrManager &Instance::Get(void)
+{
+    return mThreadNetif.mMlrManager;
+}
+#endif // (OPENTHREAD_MTD || OPENTHREAD_FTD) && OPENTHREAD_CONFIG_MLR_ENABLE
```

Comment:
nit: do not need closing comment for small blocks
```suggestion
#endif
```